NuGet.config

```diff
@@ -11,6 +11,7 @@
     <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
     <add key="dotnet3.1" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1/nuget/v3/index.json" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
+    <add key="nightly_iot_builds" value="https://pkgs.dev.azure.com/dotnet/IoT/_packaging/nightly_iot_builds/nuget/v3/index.json" />
```

Comment:
LGTM, should we also update `dotnet5` nuget feed to `dotnet6`?

Reply:
I believe you are right. This will be taken care of by arcade automatically the next time that we take on a serviced dependency, so I'll let arcade update it when it's time.